service-tests/crates.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'crates', title: 'crates.io' });
+module.exports = t;
+
+t.create('license')
+  .get('/l/libc.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('license'),
+    value: Joi.not('undefined')
```

Comment:
Could you just use `'MIT/Apache-2.0'` as before? Or, if for some reason this package's license is likely to change, choose a different one?

Reply:
Done! I don't anticipate the license to change, since the dual MIT/Apache-2.0 license is pretty standard across Rust projects, and `libc` is currently the most download crate.